### DISCUSSION TITLE
[Feature Request] Project level documentation

### DISCUSSION THREAD

**Discussion by ottaviane** [wintercms/winter#259] [Ideas]
Hi all,
why don't provide a sectione named for example 'NOTES' where developers can store and create the tecnical documentation of the site? Documentation could contain formatted text, images, charts, tables ecc.
and it will appear as a helpfull documentation for the maintenance... for example.
What do you think?
Hi.

**Comment by LukeTowers** [comment]
@ottaviane that's the longer term goal of the https://github.com/wintercms/wn-docs-plugin

**Reply by ottaviane** [reply]
wow thanks. I go to install it. Bye

**Reply by LukeTowers** [reply]
Right now it only includes the core documentation. Automated API generation and an API for other developers to register non-API docs for their plugins is being worked on, and then eventually the goal is to add a section to read and write project level documentation.

**Reply by ottaviane** [reply]
ok. thank you very mutch.
